src/calls/admin.js

```diff
@@ -61,7 +61,7 @@ export const getAllEmployeesWithRoles = async (
 
 export const setRoles = async (employeeId, roles, error) => {
   try {
-    await fetch(`${process.env.REACT_APP_API}/api/v3/role/${employeeId}`, {
+    await fetch(`${process.env.REACT_APP_API}/api/v1/employee/${employeeId}/roles`, {
       method: 'post',
```

Comment:
Please update endpoint to '/api/v1/employees/${employeeId}/roles'. It would be more logic this way